src/HTML_Renderer/org/lobobrowser/html/style/StyleSheetRenderState.java

```diff
@@ -392,10 +392,12 @@ public int getTextTransform() {
     return tt;
   }
 
+  @SuppressWarnings("deprecation")
   public final FontMetrics getFontMetrics() {
     FontMetrics fm = this.iFontMetrics;
     if (fm == null) {
       // TODO getFontMetrics deprecated. How to get text width?
+      // sugestion: using LineMetrics instead 
```

Comment:
Yes that's a good suggestion. Please create an issue for this and remove the comment and `SuppressWarnings` here.